src/common/grouping.c

```diff
@@ -82,6 +82,7 @@ int dt_grouping_remove_from_group(const int image_id)
     wimg->group_id = image_id;
     dt_image_cache_write_release(darktable.image_cache, wimg, DT_IMAGE_CACHE_SAFE);
     imgs = g_list_prepend(imgs, GINT_TO_POINTER(image_id));
+    imgs = g_list_prepend(imgs, GINT_TO_POINTER(img_group_id));
```

Comment:
Can you explain this one ? I don't understand the use!

Especially since the signal is only caught in thumtable and calls `_dt_image_info_changed_callback` where we iterate over all values comparing to the imgid.

Reply:
One part of the bug is when one removes the second image image of a group of two. The main image remains with the group symbol. 
Then this line forces the refresh of thumbnail info for the remaining image.

> specially since the signal is only caught in thumtable and calls `_dt_image_info_changed_callback` where we iterate over all values comparing to the imgid.

yes, but only on the provided list (and not all the displayed images in lt).

Reply:
Ok, maybe a small comment to clarify this in the code ?

Reply:
And maybe while we are at it only register the img_group_id if different than the image_id:

`if(image_id != img_group_id) imgs = g_list_prepend(...);`

Reply:
> And maybe while we are at it only register the img_group_id if different than the image_id:
> 
> `if(image_id != img_group_id) imgs = g_list_prepend(...);`

That's already the case, line 50...  :)


> Ok, maybe a small comment to clarify this in the code ?

I'll do it.

Reply:
Done